client/src/test/resources/valid2020/EPUB/C00000-10-chapter.xhtml

```diff
@@ -108,7 +108,7 @@
         <link rel="prev" href="C00000-11-conclusion.xhtml"/>
     </head>
     <body epub:type="bodymatter chapter" id="poem-part-chapter">
-        <h1>Chapter in poem part</h1>
+        <h2>Chapter in poem part</h2>
         <p>Should be a separate file in EPUB.</p>
     </body>
```

Comment:
Is this valid? I think all content documents are required to have a single `section` element as a child of the `body` element. So this should probably produce a validation error?

The 2020-1 guidelines says under "[3.1 Structural Divisons and Semantics](https://format.mtm.se/nordic_epub/2020-1/#structural-divisons-and-semantics)":

> The structural divisions of the publication are required to be semantically inflected by wrapping every structural part of the main text body in <section> elements, with proper nesting of subsections.


This is based on the Daisy KB. It mentions parts here (possibly in other articles as well): https://kb.daisy.org/publishing/docs/html/sections.html#faq

I think it should be:

```html
    <body>
        <section epub:type="bodymatter chapter" id="poem-part-chapter">
            <h2>Chapter in poem part</h2>
            <p>Should be a separate file in EPUB.</p>
        </section>
    </body>
```